src/lib/project.js

```diff
@@ -40,6 +40,7 @@ export async function project({ name, ui }) {
     shell.exit(1);
   }
 
+  const lang = 'ts';
```

Comment:
Just a clarifying question for me, does it make sense to support a JavaScript example given o1js is expected to be used by TypeScript only? Even if only the UI is in JS, they would need to configure TS to build the contracts and include the build files manually inside their ui project. Does it make sense to only provide TS as the default option? What do you think?

Reply:
I agree, I added it just to reflect the examples part. But in general I think it is legacy leftovers, maybe there were some ideas back then. 
cc @ymekuria if you can shed some light on this one.

Reply:
I agree with you all here. This was a legacy leftover that doesn't makes sense imo.

Reply:
Variables were removed and methods now have default parameters values.